backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5652840_sys_gh13497_IntroduceIsBPartnerInvoicesWithVendorsFlagOnWarehouse.sql

```diff
@@ -0,0 +1,201 @@
+;
+
+-- 2022-08-24T15:52:09.862Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(581380) 
+;
+
+-- 2022-08-24T15:52:09.865Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=705529
+;
+
+-- 2022-08-24T15:52:09.873Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(705529)
+;
+
+-- UI Element: Lager und Lagerort(139,D) -> Lager(177,D) -> main -> 20 -> flags.External BPartner
+-- Column: M_Warehouse.IsExternalBPartner
+-- 2022-08-24T15:54:03.892Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_Element_ID,AD_UI_ElementGroup_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayed_SideList,IsDisplayedGrid,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNo_SideList,SeqNoGrid,Updated,UpdatedBy) VALUES (0,705529,0,177,612204,540174,'F',TO_TIMESTAMP('2022-08-24 18:54:03','YYYY-MM-DD HH24:MI:SS'),100,'Y','N','N','Y','N','N','N',0,'External BPartner',110,0,0,TO_TIMESTAMP('2022-08-24 18:54:03','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- Field: Lager und Lagerort(139,D) -> Lager(177,D) -> External BPartner
+-- Column: M_Warehouse.IsExternalBPartner
+-- 2022-08-24T15:54:46.453Z
+UPDATE AD_Field SET DisplayLogic='',Updated=TO_TIMESTAMP('2022-08-24 18:54:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Field_ID=705529
+;
+
+-- Column: M_Warehouse.IsExternalBPartner
+-- 2022-08-24T15:55:49.137Z
+UPDATE AD_Column SET IsLazyLoading='N',Updated=TO_TIMESTAMP('2022-08-24 18:55:49','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=584220
+;
+
+-- Column: M_Warehouse.IsExternalBPartner
+-- Column SQL (old): (SELECT NOT EXISTS(SELECT 1 from ad_orginfo where ad_orginfo.org_bpartner_id = m_warehouse.c_bpartner_id AND ad_orginfo.ad_org_id = m_warehouse.ad_org_id))
+-- 2022-08-24T16:00:02.556Z
+UPDATE AD_Column SET ColumnSQL='(case when (SELECT NOT EXISTS(SELECT 1 from ad_orginfo where ad_orginfo.org_bpartner_id = m_warehouse.c_bpartner_id AND ad_orginfo.ad_org_id = m_warehouse.ad_org_id)) then ''Y'' else ''N'' end)', IsLazyLoading='Y',Updated=TO_TIMESTAMP('2022-08-24 19:00:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=584220
+;
+
+-- UI Element: Lager und Lagerort(139,D) -> Lager(177,D) -> main -> 20 -> flags.External BPartner
+-- Column: M_Warehouse.IsExternalBPartner
+-- 2022-08-24T16:00:56.758Z
+UPDATE AD_UI_Element SET IsDisplayed='N',Updated=TO_TIMESTAMP('2022-08-24 19:00:56','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_UI_Element_ID=612204
```

Comment:
i'm missing the descriptions that i proposed in the issue..or any descriptions for "Invoices with suppliers" :-)